### ISSUE TITLE
[FEATURE] Improve web accessibility based on WCAG

### ISSUE THREAD

**Issue by samc18** [samc18/my-finances-checker#12]
**Describe the solution you'd like**
Improve the web accessibility of the project.

Items to consider:
- [ ] Low contrast
- [ ] Missing alt text for images
- [ ] Empty links
- [ ] Missing form input labels
- [ ] Empty buttons
- [ ] Missing document language
- [ ] Make sure all links are accessible with the keyboard.

If any of the items are in accordance with the WCAG requirements there is no need to make changes.

This article can help with insights on web accessibility: https://a11y.coffee/quick-wins/
